vxyowsup/whatsapp.py

```diff
@@ -107,9 +115,19 @@ def _kill():
         self.stack.execDetached(_stop)
         self.stack.execDetached(_kill)
 
+    def send_to_stack(self, text, toAddress):
+        # encapsulate text and address?
+        # rather have it interact with stack only?
+        self.stack.execDetached(self.echo_layer.send, text, toAddress)
```

Comment:
The method on `echo_layer` appears to be called `send_to_human` rather than just `send`? I like `send_to_human`, so we should update the reference here. :)